### ISSUE TITLE
F contrasts are not to be passed forward

### ISSUE THREAD

**Issue by effigies** [bids-standard/stats-models#25]
Decided: Only t-tests and untested weighted combinations are available as input contrasts to the next level. i.e, these are terminal tests.

@jmumford To be clear, when used as an equivalent to 2-sided t-tests, does this still apply?

**Comment by jmumford** [comment]
Yes, if they'd want to pass that contrast to a higher level they'd need to specify as a t contrast as well as the f-stat if they want 2-sided.  

If that's too clunky, let me know and I can try to think of other solutions (although I think it would only be to allow 1 and 2-sided t-tests as well as f-tests, which I thought might be more difficult to implement).

**Comment by effigies** [comment]
Nah, it's fine for v1. Thanks!
